src/app.py

```diff
@@ -91,20 +91,34 @@ def bootstrap():
                             res = requests.get(url=master_url + '/htmlreport')
                             with open(os.path.join(report_path, 'reports.html'), "wb") as file:
                                 file.write(res.content)
-                            logger.info('Reports is successfully downloaded.')
+                            logger.info('Reports have been successfully downloaded.')
                         else:
                             logger.error('Locust cannot be started. Please check logs!')
 
                         break
                     else:
-                        logger.error('Attempt: {attempt}. Locust master might not ready yet. '
+                        logger.error('Attempt: {attempt}. Locust master might not ready yet.'
                                      'Status code: {status}'.format(attempt=_, status=res.status_code))
             except ValueError as v_err:
                 logger.error(v_err)
 
+    elif role == 'local':
+        automatic = convert_str_to_bool(os.getenv('AUTOMATIC', str(False)))
+        logger.info('Automatic run: {auto}'.format(auto=automatic))
+        if not os.environ.get('MASTER_HOST'): os.environ["MASTER_HOST"] = '127.0.0.1'
+        os.environ["ROLE"] = 'master'
```

Comment:
using for loop looks better in my opinion:
`for role in ['master', 'slave', 'controller’]: 
      os.environ['ROLE'] = role  
      bootstrap(1)
`